src/js/a11y/020-tabbing.js

```diff
@@ -24,7 +24,7 @@ AddEvent( document.querySelector( '.js-tabbing-switch' ), 'click', function( eve
 /**
  * Event listener for when the user scrolls or uses the tab key to stop Tab() function
  */
-AddEvent( document.querySelector( '.tabbing-frame' ), 'keydown', function( event, $this ) {
+ AddEvent( document.querySelector( '.tabbing-frame' ), 'keydown', function( event, $this ) {
```

Comment:
Can you click on the button below here and accept the change?
```suggestion
AddEvent( document.querySelector( '.tabbing-frame' ), 'keydown', function( event, $this ) {
```